src/tqec/circuit/measurement_map.py

```diff
@@ -8,8 +8,8 @@ class used to represent and query measurement offsets in a circuit.
 
 from __future__ import annotations
 
+from collections.abc import Sequence
```

Comment:
Is there an explanation for not using `typing` and using `collections.abc`?

Reply:
Because `ruff` prefers `collections.abc` over `typing`.